### ISSUE TITLE
is big sur not compatiable?

### ISSUE THREAD

**Issue by hypdboy** [Anjok07/ultimatevocalremovergui#1057]
hi im using big sur, its the highest os my mac can run but UVR wont open. this is what is says in terminal, i dont understand code or anything but i think it means my os is too outdated

Reason: Incompatible library version: _lzma.cpython-311-darwin.so requires version 10.0.0 or later, but liblzma.5.dylib provides version 8.0.0

**Comment by ChrisTG742** [comment]
Have you tried the stand-alone binary installer from Python? For example: https://www.python.org/ftp/python/3.11.5/python-3.11.5-macos11.pkg
If that installs 3.11 without moaning, the problem lies somewhere else.

**Comment by hypdboy** [comment]
yes, that installed fine thank you, but im not really sure what to do with it now

**Comment by ChrisTG742** [comment]
I think running UVR by the just installed version of Python should do the trick. Unfortunately I don't have a Mac here to test or try out.
@Anjok07 could you please jump in? Or any other Mac-pro here?
